### ISSUE TITLE
Deal with overfitting

### ISSUE THREAD

**Issue by yangtcai** [yangtcai/Ensembl-Repeat-Identification#34]
Hi, @williamstark01, I test two different hyperparameter to deal with overfitting.The orange one is the first time I trained our model, it's learning rate 0.001, transformer encoder-decorder 6 layers. When I find our model is overfitting, I  find reletived issue in DETR, small datasets could lead to this problem, so I change the layers from 6 to 3. The blue one's learning rate is 0.001, transformer encoder-decorder 3 layers. And the red one, I change the dropout from 0.2 to 0.1, and also change the learning rate to 0.0001.
<img width="295" alt="image" src="https://user-images.githubusercontent.com/77475012/175893817-aefd8883-3094-4fa0-a668-d9169b97a8ce.png">
<img width="323" alt="image" src="https://user-images.githubusercontent.com/77475012/175893864-f3c6b6a5-5c02-4893-a543-b10ca1cb792c.png">
The related issue link: https://github.com/facebookresearch/detr/issues/342
I think we can add more chromosome datasets into train our model, as COCO datasets are up to 330k, we only have 8.9k to train. Am I understanding is correct?

**Comment by williamstark01** [comment]
Almost forgot:

> I think we can add more chromosome datasets into train our model, as COCO datasets are up to 330k, we only have 8.9k to train. Am I understanding is correct?

This is correct, the training set is relatively small at this point. We should get better results if we add more chromosomes in the dataset. New issue: #36

**Comment by yangtcai** [comment]
>At this stage it would be good to organize how we track multiple training experiments. TensorBoard is a good option, we just need to also save hyperparameter values that will help us filter experiments. Could you share the one you are currently using?
Hi, @williamstark01, I'm slightly confused with this part, do you mean the hyperparameters or TenserBoard files? :D

**Comment by williamstark01** [comment]
I meant the URL to the TensorBoard dashboard if you are using the official public one ( https://tensorboard.dev/ ) and uploading the log there. Or are you running a TensorBoard locally?
